src/Pages/Products/ProductDetails/SlideCarousel.js

```diff
@@ -0,0 +1,117 @@
+import React, { useState, useEffect } from "react";
+import styled, { css } from "styled-components";
+import Slide from "./Slide";
+import Thumbnail from "./Thumbnail";
+
+const SlideCarousel = () => {
+  const [optionData, setOptionData] = useState([]);
+
+  useEffect(() => {
+    fetch("/data/data.json")
+      .then((res) => res.json())
+      .then((res) => setOptionData(res.option));
+  }, []);
+
+  const [currentSlide, setCurrentSlide] = useState(0);
+  const [positionX, setPositionX] = useState(0);
```

Comment:
useState 문들은 위해 한곳에 다 모아주시는게 추후에 관리하기에 편리합니다.
```suggestion
  const [optionData, setOptionData] = useState([]);

  useEffect(() => {
    fetch("/data/data.json")
      .then((res) => res.json())
      .then((res) => setOptionData(res.option));
  }, []);

  const [currentSlide, setCurrentSlide] = useState(0);
  const [positionX, setPositionX] = useState(0);
```
```suggestion
  const [optionData, setOptionData] = useState([]);
  const [currentSlide, setCurrentSlide] = useState(0);
  const [positionX, setPositionX] = useState(0);

  useEffect(() => {
    fetch("/data/data.json")
      .then((res) => res.json())
      .then((res) => setOptionData(res.option));
  }, []);
```